### PR TITLE
Fixes 3171: add gpg key path for red hat repos

### PR DESCRIPTION
**Pull request by rverdile**
## Summary
The repository config file wasn't providing the correct gpg key path for red hat repos. All the red hat repos have the same gpg key path, so this hard codes that value. 

## Testing steps
1. Import red hat repos and create a snapshot
2. Fetch the config.repo file from the `/repositories/:uuid/snapshots/:snap_uuid/config.repo/` endpoint
3. The config.repo file should have the red hat gpg key path

## Checklist

- [x] Tested with snapshotting feature disabled and pulp server URL not configured if appropriate

### CODE REVIEW

File: pkg/dao/repository_configs_test.go [hunk 1]
```diff
@@ -1554,6 +1554,8 @@ func (suite *RepositoryConfigSuite) TestBulkDeleteRedhatRepository() {
 	orgID := config.RedHatOrg
 	repoConfigCount := 5
 
+	suite.tx.Exec("TRUNCATE repositories, snapshots, repositories_rpms, repositories_package_groups, repository_configurations")
```
Comment (jlsherrill) [inline pkg/dao/repository_configs_test.go:1557]: any idea why this is needed?

Comment (jlsherrill) [reply pkg/dao/repository_configs_test.go:1557]: i guess i should rephrase it, can we fix the test so this isn't needed?   I get its in a transaction and gets rolled back, so maybe its not a big deal, but it feels really bad

Comment (rverdile) [reply pkg/dao/repository_configs_test.go:1557]: that's fair. I can change that

**Comment by jlsherrill** [comment]
https://issues.redhat.com/browse/HMS-3171

File: pkg/dao/repository_configs_test.go (hunk 1 above)
Comment (rverdile) [reply pkg/dao/repository_configs_test.go]: fixed

**Review by jlsherrill — APPROVED** [review]

**Pull request merged**